src/libxively/sft/xi_sft_logic.c

```diff
@@ -7,6 +7,10 @@
 #include <xi_sft_logic.h>
 #include <stddef.h>
 #include <xi_macros.h>
+#include <xi_debug.h>
+
+// #define XI_SFT_DOWNLOAD_BYTES_PER_FILE_CHUNK 1 * 1024
```

Comment:
We already have other configuration options (the max cbor size).  I would put them all in a header file somewhere related to SFT and then do #ifndef around it so that it COULD be defined in the makefile, if desired.